hadoop-ozone/dev-support/checks/unit.sh

```diff
@@ -17,7 +17,7 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 cd "$DIR/../../.." || exit 1
 
 export MAVEN_OPTS="-Xmx4096m"
-mvn -B -DskipShade -Dskip.yarn -fae test -pl \!:hadoop-ozone-integration-test "$@"
+mvn -B -DskipShade -Dskip.yarn -fae install -pl \!:hadoop-ozone-integration-test "$@"
```

Comment:
Can you please explain the goal of this change?

Reply:
I tried to understand why the unit tests are failed and I was afraid that some changes are missing because the missing install phase. But I think it was because the tricky provided dependency between hdfs server and hdfs.

I reverted the change.